docs/UserGuide.md

```diff
@@ -125,6 +129,17 @@ This guide serves to:<br>
 * An example is shown below:
 ![SortExample](images/SortExample.png)
 
+#### Responsive UI when resizing window
+
+* The UI will change dynamically depending on the window width.
+
+* If the window width is too small, WorkBook will switch to a narrow layout for a better UX.
+
+* Otherwise, WorkBook will be in the wide layout which has an extra right panel for displaying tips.
+
+    ![ResponsiveUI](images/ResponsiveUi.gif)
```

Comment:
might want to add a picture under each condition on how it'll look like

Reply:
I've added the side by side pictures from the getting started section.